### DISCUSSION TITLE
How can I browse my Copilot Chat history in the Jetbrains UI?

### DISCUSSION THREAD

**Discussion by codekiln** [community/community#101999] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Prompt-crafting plays an important part in the success of ai-assisted interactions, and those prompts, once crafted, are a form of meta-programming source code. To remove or conceal meta source code without the code author's consent is not substantively different from removing or concealing source code without the code author's consent. Imagine an IDE that lacked the ability to save to disk, that's what copilot chat feels like right now. In my opinion, being able to 1. save past chats and 2. search them are fundamental, foundational features with AI-assisted interactions, every bit as much as the ability to save code files to disk and search them.

After searching around on google, I couldn't find out whether either of these features were implemented or planned. I couldn't find history in the JetBrains Copilot Chat UI, or determine if it was a planned but as-yet-unimplemented feature in JetBrains, if the copilot team had decided not to include this feature, etc.

Please make it very clear what this product's relationship is to chat history so as to decrease confusion.

**Comment by dusnoki** [comment]
Would like to know as well.

**Comment by robertg042** [comment]
Probably we can't which is a shame

@codekiln actually, just found out they are working on it (source: https://github.com/orgs/community/discussions/69250?sort=new#discussioncomment-8396771)
